presto-hive/src/main/java/io/prestosql/plugin/hive/HiveTableProperties.java

```diff
@@ -123,6 +125,8 @@ public HiveTableProperties(TypeManager typeManager, HiveConfig config)
                         config.getOrcDefaultBloomFilterFpp(),
                         false),
                 integerProperty(BUCKET_COUNT_PROPERTY, "Number of buckets", 0, false),
+                integerProperty(TEXT_SKIP_HEADER_COUNT, "Number of header lines", 0, false),
```

Comment:
nit. I think default should be null to distinguish case when user set it explicitly to case when user is using default.